src/static-pages/components/JournalPolicies.tsx

```diff
@@ -1,11 +1,14 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import Interweave from 'interweave';
 import { useTranslation } from 'react-i18next';
 import { Paragraph } from '../../ui/atoms';
+import scrollToAnchor from '../utils/scrollToAnchorEffect';
 
 const JournalPolicies = (): JSX.Element => {
     const { t } = useTranslation('author-guide');
 
+    useEffect(scrollToAnchor);
```

Comment:
We should follow the pattern used in ui [here](https://github.com/libero/reviewer-client/tree/master/src/ui/hooks) and turn this into a `useScrollToAnchor` [custom hook](https://reactjs.org/docs/hooks-custom.html).

Reply:
Thanks for the pointer, makes a lot of sense. Have updated the code. Note though that like some of the other custom hooks it doesn't seem to entirely fit having a unit test. Instead though the browser tests should be updated to test this functionality if we want to go live with it..